include/nanorange/views/empty.hpp

```diff
@@ -24,9 +24,6 @@ class empty_view : view_interface<empty_view<T>> {
     static constexpr T* data() noexcept { return nullptr; }
 
     static constexpr bool empty() noexcept { return true; }
-
-    friend constexpr T* begin(empty_view) noexcept { return nullptr; }
-    friend constexpr T* end(empty_view) noexcept { return nullptr; }
```

Comment:
Not really a constructor, but I thought this was small enough that it would be fine to go alone with the other changes. Seems like these two were dropped from the draft.

https://eel.is/c++draft/range.empty

Reply:
They're no longer needed after the `borrowed_range` changes, good catch.